frontend/src/screens/Map/TimeSeriesSlider.js

```diff
@@ -80,22 +80,40 @@ const debounceSourceChange = debounce(changeModifiedLayerSource, 200);
 const TimeSeriesSlider = ({ availableDates, modifiedLayer }) => {
   const marks = [];
   const labels = [];
+  const { dataDate } = availableDates[0];
```

Comment:
I would move this into the for cycle and iterate it, `let { dataDate } = availableDates[i];` in case some entry doesn't have dataDate

Reply:
Fixed:
@srezacova
Show date if dataDate does not exist on timeline
47f99f8